### PR TITLE
ZWOO-37: bug fixes

### PR DESCRIPTION
**Pull request by fabiankachlock**
fixes:
- ZWOO-118: scenarios in which the hosts disconnects
- ZWOO-112: notify the player when he gets host
- ZWOO-119: selecting the correct player after kicking one ingame

### CODE REVIEW

File: backend/ZwooBackend/Controllers/GameController.cs [hunk 1]
```diff
@@ -91,6 +91,7 @@ public IActionResult JoinGame([FromBody] JoinGame body)
                         return Unauthorized(ErrorCodes.GetErrorResponseMessage(ErrorCodes.Errors.INVALID_PASSWORD, "wrong password"));
                     } 
                     else
+
```
Comment (Visual-Rock) [inline backend/ZwooBackend/Controllers/GameController.cs]: ?

File: frontend/src/locales/de.json [hunk 2]
```diff
@@ -319,7 +319,8 @@
       "spectatorLeft": "{0} hat das Spiel verlassen",
       "changedRoleToHost": "{0} ist nun Host",
       "changedRoleToPlayer": "{0} ist nun Spieler",
-      "changedRoleToSpectator": "{0} ist nun Zuschauer"
+      "changedRoleToSpectator": "{0} ist nun Zuschauer",
+      "selfGotHost": "Du bist nun Host"
```
Comment (Visual-Rock) [inline frontend/src/locales/de.json:323]: Host -> Veranstalter

**Review by Visual-Rock — APPROVED** [review]
✔️

File: frontend/src/locales/de.json (hunk 2 above)
Comment (fabiankachlock) [reply frontend/src/locales/de.json:323]: NEEEEEEEEIN

**Pull request merged**